README.md

```diff
@@ -16,6 +16,7 @@ You'll need these things to build the project:
 
  - Python 3
  - Ninja
+ - Yarn
```

Comment:
Yarn isn't required if you just want to run the emulator independently of the app for debugging.
```suggestion
 - Yarn (only when building for iOS)
```

Reply:
Amended.